eduHub/src/main/kotlin/com/arsiu/eduhub/protobuf/handlers/assignment/FindByIdHandler.kt

```diff
@@ -0,0 +1,42 @@
+package com.arsiu.eduhub.protobuf.handlers.assignment
+
+import com.arsiu.eduhub.mapper.AssignmentNatsMapper
+import com.arsiu.eduhub.model.Assignment
+import com.arsiu.eduhub.service.AssignmentService
+import com.arsiu.eduhub.v2.assignmentsvc.commonmodels.assignment.AssignmentRequest
+import com.arsiu.eduhub.v2.assignmentsvc.input.reqreply.assignment.FindByIdAssignmentRequest
+import com.arsiu.eduhub.v2.assignmentsvc.output.reqreply.assignment.FindByIdAssignmentResponse
+import org.springframework.stereotype.Component
+import reactor.core.publisher.Mono
+
+@Component
+class FindByIdHandler(
+    private val service: AssignmentService,
+    private val mapper: AssignmentNatsMapper
+): AssignmentHandler<FindByIdAssignmentRequest, FindByIdAssignmentResponse>(){
```

Comment:
```suggestion
) : AssignmentHandler<FindByIdAssignmentRequest, FindByIdAssignmentResponse>() {
```

please, apply formatting to all files

Reply:
`+`